src/base58.cpp

```diff
@@ -84,44 +90,47 @@ static const int8_t mapBase58[256] = {
     return true;
 }
 
+auto ConvertToLongs(const Span<const unsigned char>& input, int start) -> std::vector<int64_t> {
+    int effectiveLength = input.size() - start;
+    std::vector<int64_t> inputAsLongs((effectiveLength + (encodeGroupSize - 1)) / encodeGroupSize);
```

Comment:
this is basically a ceil integer division, i.e. `(int) ceil(effectiveLength / (double) encodeGroupSize)`